models/Event.js

```diff
@@ -0,0 +1,65 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const EventSchema = new Schema({
+  user: {
+    type: Schema.Types.ObjectId,
+    ref: 'users',
+  },
+  title: {
+    type: String,
+    required: true,
+  },
+  description: {
+    type: String,
+    required: true,
+  },
+  dateStart: {
+    type: Date,
+    default: Date.now,
+    required: true,
+  },
+  dateEnd: {
+    type: Date,
+    required: true,
+  },
+  type: {
+    type: String,
+    required: true,
+  },
+  location: {
+    type: String,
+    required: true,
+  },
+  website: {
+    type: String,
+  },
+  image: {
+    type: String,
+  },
+  participants: [
+    {
+      name: {
+        type: String,
+        required: true,
+      },
+      phone: {
+        type: String,
+        required: true,
+      },
+      shortId: {
+        type: String,
+      },
+      status: {
+        type: String,
+        default: 'pending',

```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/spumony/xvent-project-backend/pullRequest?prid=7531905)
```suggestion
        default: "pending",
```